docs/sources/flow/reference/components/loki.source.file.md

```diff
@@ -0,0 +1,96 @@
+---
+aliases:
+- /docs/agent/latest/flow/reference/components/loki.source.file
+title: loki.source.file
+---
+
+# loki.source.file
+
+`loki.source.file` reads log entries from files and forwards them to other
+`loki.*` components.
+
+Multiple `loki.source.file` components can be specified by giving them
+different labels.
+
+## Usage
+
+```river
+loki.source.file "LABEL" {
+  targets    = TARGET_LIST
+  forward_to = RECEIVER_LIST
+}
+```
+
+## Arguments
+The component starts a new reader for each of the given `targets` and fans out
+log entries to the list of receivers passed in `forward_to`.
+The list of arguments that can be used to configure the block is presented
+below.
+
+Name         | Type                   | Description          | Default | Required
+------------ | ---------------------- | -------------------- | ------- | --------
+`targets`    | `list(map(string))`    | List of files to read from. | | yes
+`forward_to` | `list(chan loki.Entry)` | List of receivers to send log entries to. | | yes
```

Comment:
Should we alias this to a more generic type name like `LogsReceiver`? Or leave it as-is for now?

Reply:
I'd say use something generic to reduce the chances that someone asks what something like `chan loki.Entry` means.